docs/cases/cases-overview.asciidoc

```diff
@@ -68,3 +69,32 @@ To view a case, click on its name. You can then:
 * Refresh the case to retrieve the latest updates
 
 NOTE: Comments can also contain Markdown syntax and Timeline links.
+
+[float]
+[[cases-lens-visualization]]
+== Add a Lens visualization
+
+beta[]
+
+Add a Lens visualization to your case to portray event and alert data through charts and graphs.
+
+IMPORTANT: Set an absolute time range for your visualization. This ensures your visualization doesn't change over time after you save it to your case and provides important context for other viewers.
+
+[role="screenshot"]
+image::images/add-vis-to-case.gif[Shows how to add a visualization to a case]
+
+To add a Lens visualization to your case within a comment:
```

Comment:
This could be misread to say that the case is within a comment. Maybe something like this? Or add a step at the beginning of the procedure to create/open the comment?
```suggestion
To add a Lens visualization to a comment within your case:
```